Content.Shared/Wires/SharedWiresSystem.cs

```diff
@@ -132,4 +137,21 @@ public bool IsPanelOpen(Entity<WiresPanelComponent?> entity)
 
         return entity.Comp.Open;
     }
+
+    private void OnAttemptOpenActivatableUI(EntityUid uid, ActivatableUIRequiresPanelComponent component, ActivatableUIOpenAttemptEvent args)
+    {
+        if (args.Cancelled || !TryComp<WiresPanelComponent>(uid, out var wires))
+            return;
+
+        if (component.RequireOpen != wires.Open)
```

Comment:
This looked wrong until I looked at the doc for RequireOpen.

RequireOpen really should be renamed to something that matches it's intended usage better.